crm_compassion/models/weekly_demand.py

```diff
@@ -136,6 +154,34 @@ def _default_unsponsored_ambassador(self):
             'crm_compassion.number_children_ambassador'))
         return allocate_per_week - (float(ambass_sponsored) / 52.0)
 
+    @api.multi
+    def _compute_resupply_sub(self):
+        """ Compute SUB resupply. """
+        subs = list()
```

Comment:
instead of a list, you can simply use an integer to count how many sponsorships are sub rejects.